EXAMPLES/LuoYang_fluid_solid_kernel/compare_two_images_to_compare_sensitivity_kernels.py

```diff
@@ -0,0 +1,92 @@
+#! /usr/bin/python
+
+# taken from http://www.pyimagesearch.com/2014/09/15/python-compare-two-images/
+# in April 2015
+
+# computes the Mean Squared Error (MSE) and the Structural Similarity Index (SSIM)
+# between two images (having the exact same size)
+
+# Slightly modified by Dimitri Komatitsch, April 2015
+
+############################################################################
+# requires the "python-skimage" and "python-opencv" packages to be installed
+############################################################################
+
+# In 3D comparing kernels i.e. volumetric stuff would be a bit difficult, however in 2D we can see them as images if we draw them and then we can use Python routines that compute a similarity index between two images.
+#
+# I took one from http://www.pyimagesearch.com/2014/09/15/python-compare-two-images/ and slightly modified it. It works well. You then just need to check if the Structural Similarity Index (SSIM) is greater than 0.99.
+#
+# For the 2D code we can apply this to EXAMPLES/LuoYang_fluid_solid_kernel for instance in the nightly builds.
+#
+# For the 3D code for now we could just draw pictures along a couple of cut planes and use that script as well. Better than nothing.
+
+# import the necessary packages
+from skimage.measure import structural_similarity as ssim
+import matplotlib.pyplot as plt
+import numpy as np
+import cv2
```

Comment:
This is a pretty heavy dependency. It's also probably unnecessary. You can read PNG images using `matplotlib.image.imread`.